apps/ledger-live-desktop/src/renderer/components/MainSideBar/index.tsx

```diff
@@ -370,6 +386,9 @@ const MainSideBar = () => {
               <IconChevron size={16} />
             </Collapser>
             <SideBarScrollContainer>
+              {ldmkTransportFlag?.enabled && (
+                <LDMKTransportFlag>With LDMK transport</LDMKTransportFlag>
+              )}
```

Comment:
[COULD] maybe add a param `warningVisible: boolean` in the feature flag so that we can hide this if we want to enable it in prod one day